201-vmss-windows-vm-largescale-with-managed-disk/azuredeploy.parameters.json

```diff
@@ -0,0 +1,6 @@
+{
+    "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {	
```

Comment:
Can we fill the parameters in this file too?

Reply:
Done. Noticed I wasn't able to  include vmssName var as its default value uses a function.